app/models/profile.rb

```diff
@@ -60,4 +79,14 @@ def custom_profile_attributes
   def clear!
     update(data: {})
   end
+
+  private
+
+  def conditionally_validate_summary
+    return unless ProfileField.exists?(attribute_name: SUMMARY_ATTRIBUTE) && summary.present?
+    # Grandfather in people who had a too long summary before.
+    return if data_was[SUMMARY_ATTRIBUTE] && data_was[SUMMARY_ATTRIBUTE].size > MAX_SUMMARY_LENGTH
```

Comment:
TIL about `data_was` !

Reply:
The `data` part is just the name of the column, you can always access the previous value in an update action as `column_name_was`.